documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -803,28 +803,31 @@ Only alphanumeric characters and underscores should be used.
 |`false`
 |When SSL is enabled this setting controls whether strict hostname checking is disabled during connection phase. If `true` the connection will not prevent man-in-the-middle attacks.
 
-|[[mongodb-property-database-whitelist]]<<mongodb-property-database-whitelist, `database.whitelist`>>
+|[[mongodb-property-database-whitelist]]<<mongodb-property-database-whitelist [[mongodb-property-database-include-list]]<<mongodb-property-database-include-list , `database.include.list`>>
```

Comment:
I always kept the old anchor links so that if there are links in external blog posts leading to the old anchor they are still working but pointing to the new property.

Reply:
+1